apps/web/pages/settings/security/two-factor-auth.tsx

```diff
@@ -32,11 +32,19 @@ const TwoFactorAuthView = () => {
 
   if (isLoading) return <SkeletonLoader />;
 
+  const isCalProvider = user?.identityProvider === "CAL";
   return (
     <>
       <Meta title={t("2fa")} description={t("2fa_description")} />
+      {!isCalProvider && (
+        <Alert
+          severity="neutral"
+          message="Two-Factor authentication can be enabled only for email and password authentication."
```

Comment:
approved but we should really translate this :)

Reply:
done :)